spot-contracts/contracts/PerpetualTranche.sol

```diff
@@ -784,8 +890,8 @@ contract PerpetualTranche is
         uint256 stdTrancheInAmt = _toStdTrancheAmt(trancheInAmt, computeDiscount(trancheIn));
         uint256 trancheInPrice = computePrice(trancheIn);
         uint256 perpAmtMint = (totalSupply_ > 0)
-            ? (stdTrancheInAmt * trancheInPrice * totalSupply_) / _reserveValue()
-            : (stdTrancheInAmt * trancheInPrice) / UNIT_PRICE;
+            ? (stdTrancheInAmt * trancheInPrice).mulDiv(totalSupply_, _reserveValue())
```

Comment:
I wonder, is it safer to push as much into the mulDiv as possible? (From an overflow perspective)

So, `a.mulDiv(b * c, d)`, rather than `(a * b).mulDiv(c, d)`

Reply:
`stdTrancheInAmt` and `totalSupply_` should be at most the same order and I'd imagine `totalSupply_ > stdTrancheInAmt` most of the time.  So `trancheInPrice * totalSupply_` has more chance of overflowing than `stdTrancheInAmt * trancheInPrice`?